### PR TITLE
Adds new plugin [mlamberts78/weather-chart-card]

### PR DESCRIPTION
**Pull request by mlamberts78**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [ ] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: <https://github.com/mlamberts78/weather-chart-card/releases/tag/V2.0>
Link to successful HACS action (without the `ignore` key): <https://github.com/mlamberts78/weather-chart-card/actions/runs/6668657867>

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -619,6 +619,7 @@
   "mikelawrence/senseme-hacs",
   "mill1000/midea-ac-py",
   "MislavMandaric/home-assistant-vaillant-vsmart",
+  "mlamberts78/weather-chart-card",
```
Comment (ludeeus) [inline integration:622]: This is not an integration.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (mlamberts78) [reply integration]: Indeed chnaged the wrong file, should be plugin.

File: integration [hunk 2]
```diff
@@ -986,4 +986,4 @@
   "zeronounours/HA-custom-component-energy-meter",
   "zigul/HomeAssistant-CEZdistribuce",
   "ZsBT/hass-w1000-portal"
-]
\ No newline at end of file
+]
```
Comment (ludeeus) [inline integration:989]: This file should not be touched.
Adjust this.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 2 above)
Comment (mlamberts78) [reply integration:990]: Would you like me to create a new pull request? Will that take another 3 months?

Comment (ludeeus) [reply integration:990]: Better to clean up the current one.

**Pull request merged**